src/constants/plugins.ts

```diff
@@ -1,16 +1,16 @@
-export const HTML_REPORTER = "html-reporter/hermione";
-export const HERMIONE_TEST_REPEATER = "hermione-test-repeater";
-export const HERMIONE_RETRY_COMMAND = "hermione-retry-command";
-export const HERMIONE_REASSERT_VIEW = "hermione-reassert-view";
-export const HERMIONE_SAFARI_COMMANDS = "hermione-safari-commands";
-export const HERMIONE_RETRY_PROGRESSIVE = "hermione-retry-progressive";
-export const HERMIONE_STORYBOOK = "hermione-storybook/plugin";
-export const HERMIONE_PROFILER = "hermione-profiler";
-export const HERMIONE_HEADLESS_CHROME = "hermione-headless-chrome";
+export const HTML_REPORTER = "html-reporter/testplane";
+export const TESTPLANE_TEST_REPEATER = "@testplane/test-repeater";
+export const TESTPLANE_RETRY_COMMAND = "@testplane/retry-command";
+export const TESTPLANE_REASSERT_VIEW = "@testplane/reassert-view";
+export const TESTPLANE_SAFARI_COMMANDS = "@testplane/safari-commands";
+export const TESTPLANE_RETRY_PROGRESSIVE = "@testplane/retry-progressive";
+export const TESTPLANE_STORYBOOK = "@testplane/storybook/plugin";
+export const TESTPLANE_PROFILER = "@testplane/profiler";
+export const TESTPLANE_HEADLESS_CHROME = "@testplane/headless-chrome";
 export const URL_DECORATOR = "url-decorator";
-export const HERMIONE_TEST_FILTER = "hermione-test-filter";
-export const HERMIONE_OAUTH = "hermione-oauth";
-export const HERMIONE_PLUGINS_PROFILER = "hermione-plugins-profiler";
-export const HERMIONE_TABS_CLOSER = "hermione-tabs-closer";
-export const HERMIONE_GLOBAL_HOOK = "hermione-global-hook";
+export const TESTPLANE_TEST_FILTER = "@testplane/test-filter";
+export const TESTPLANE_OAUTH = "@testplane/oauth";
+export const TESTPLANE_PLUGINS_PROFILER = "@testplane/plugins-profiler";
+export const TESTPLANE_TABS_CLOSER = "@testplane/tabs-closer";
+export const TESTPLANE_GLOBAL_HOOK = "@testplane/global-hook";
 export const RETRY_LIMITER = "retry-limiter";
```

Comment:
Dont we plan to move this two plugins to `@testplane` scope too?

Reply:
To be honest I don't know… for consistency it would be necessary...